### PR TITLE
feat: design system checkbox page with example

### PR DESCRIPTION
**Pull request by Matgorn**
PR includes:

chore: setup tailwind config based on Selleo Design System
feat: add checkbox based on [figma](https://www.figma.com/file/aAZNLti1x7RHcKO2aaVdyh/Selleo-Design-System?node-id=1512%3A1606&t=9A4m8JXIS5wFtFrp-4) with additional error state presentation

edit: Design system on Figma includes example with only dark background so checkbox label styles are just proposition.
@michalgren I would appreciate some suggestions here for light theme.

### CODE REVIEW

**Comment by Matgorn** [comment]
Design system on Figma includes example with only dark background so checkbox label styles are just proposition.
@michalgren I would appreciate some suggestions here for light theme.

File: packages/selleo-design-core/src/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+export function CheckboxWithError() {
+  return <div class="flex items-center">
+  <input id="checkbox-error-id" type='checkbox' class='hidden peer' value='' />
+  <label htmlFor='checkbox-error-id' class='flex justify-center text-base text-white cursor-pointer before:rounded-sm before:w-2 before:h-2 before:mr-2 before:outline before:outline-2 before:outline-offset-2 before:outline-danger hover:before:outline-brand-primary-500 peer-checked:before:bg-brand-primary-500 peer-checked:before:outline-brand-primary-500'>
+    <span class='text-black dark:text-white'>I <span class='text-danger'>accept</span> the <a class='text-black dark:text-white hover:no-underline'>Privacy Policy</a></span>
+  </label>
+</div>
+}
```
Comment (k1eu) [inline packages/selleo-design-core/src/Checkbox.tsx:19]: pls configure ide to support automatically adding newlines at the end of the file

Comment (k1eu) [reply packages/selleo-design-core/src/Checkbox.tsx:19]: I guess easy fix so approved :>

**Review by k1eu — APPROVED** [review]

**Comment by Matgorn** [comment]
Update:
- Change approach with achieving the board with offset on `:checked` state from outline to board + ring
- Update checkbox styles for light theme as per [Figma Design](https://www.figma.com/file/aAZNLti1x7RHcKO2aaVdyh/Selleo-Design-System?node-id=2354%3A7637&t=tCDrOFoFz4K4JAVP-4)

**Review by k1eu — APPROVED** [review]

**Pull request merged**